arbitrum-docs/stylus/concepts/public-preview-expectations.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Public preview: What to expect'
+sidebar_label: 'Public preview'
+description: 'Stylus is currently a public preview capability. This concept document explains what this means, and what to expect from public preview capabilities.'
+author: symbolpunk
+sidebar_position: 1
+---
+
+Stylus is currently a **public preview** offering. This concept document explains what "public preview" means, what to expect from public preview capabilities, and how to engage with our team as you tinker.
```

Comment:
This introduces some duplication - most of this copy is also present in the [Orbit public preview doc](https://docs.arbitrum.io/launch-orbit-chain/concepts/public-preview-expectations). We can DRY this up post-launch.